### ISSUE TITLE
✅ 🐞 Bug Report: Mega Menu - Laptop / Small Screens - NOT Scrollable - [ED-11023]

### ISSUE THREAD

**Issue by kdub718** [elementor/elementor#22712]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The mega menu doesn't have a scroll bar vertically on small screens/windows sizes like a 13" laptop when still in a desktop mode.

### Steps to reproduce

This is a preview the mega menu I'm working on.  It's a decent size but only takes up 1/4 of the window on a 27inch imac but if I open it on my 13" laptop I lose the ability to scroll down to see the bottom of the menu.
<img width="359" alt="image" src="https://github.com/elementor/elementor/assets/19559355/19f01e31-dde4-46d4-9faa-1595df2114fa">

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt



== Server Environment ==
	Operating System: Linux
	Software: nginx
	MySQL version: Percona Server (GPL), Release '44', Revision '27f408641db' v5.7.41-44
	PHP Version: 8.0.28
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: #
	Home URL: #
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/Los_Angeles
	Admin Email: #
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.7.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==

	Elementor
		Version: 3.13.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.2
		Author: Elementor.com


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active
	Hello Theme Header &amp; Footer: Active
	Editor Top Bar: Active
	Landing Pages: Active
	Nested Elements: Active
	Lazy Load Background Images: Active
	Page Transitions: Active
	Notes: Active
	Loop: Active
	Form Submissions: Active
	Scroll Snap: Active
	Menu: Active

````

</details>

**Comment by nicholaszein** [comment]
Hello, @kdub718! 

Thank you for submitting your issue! 🙏 

We are aware of this issue and our team is already working on a fix. We can’t say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 😉

Best regards

**Comment by nicholaszein** [comment]
Hello there @kdub718!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.18.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
